packages/xrpl/snippets/src/bridgeTransfer.ts

```diff
@@ -0,0 +1,166 @@
+    Destination: wallet2.classicAddress,
+    Amount: (
+      parseInt(bridgeData.MinAccountCreateAmount as string, 10) * 2
+    ).toString(),
+  }
+  const fundResponse = await lockingClient.submitAndWait(fundTx, {
+    wallet: wallet1,
+  })
+  console.log(fundResponse)
+
+  // wait for the attestation to go through
+  let ledgersWaited = 0
+  let initialBalance = '0'
+  while (ledgersWaited < MAX_LEDGERS_WAITED) {
+    await sleep(LEDGER_CLOSE_TIME)
+    try {
+      initialBalance = await issuingClient.getXrpBalance(wallet2.classicAddress)
+      console.log(
+        `Wallet ${wallet2.classicAddress} has been funded with a balance of ${initialBalance} XRP`,
+      )
+      break
+    } catch (_error) {
+      ledgersWaited += 1
+      // eslint-disable-next-line max-depth -- needed here
+      if (ledgersWaited === MAX_LEDGERS_WAITED) {
+        throw Error('Destination account creation via the bridge failed.')
+      }
+    }
+  }
+
+  console.log(
+    `Transferring funds from ${wallet1.classicAddress} on the locking chain to ` +
+      `${wallet2.classicAddress} on the issuing_chain...`,
+  )
+
+  // Fetch the claim ID for the transfer
+  console.log('Step 1: Fetching the claim ID for the transfer...')
+  const claimIdTx: XChainCreateClaimID = {
+    TransactionType: 'XChainCreateClaimID',
+    Account: wallet2.classicAddress,
+    XChainBridge: bridge,
+    SignatureReward: bridgeData.SignatureReward,
+    OtherChainSource: wallet1.classicAddress,
+  }
+  const claimIdResult = await issuingClient.submitAndWait(claimIdTx, {
+    wallet: wallet2,
+  })
+
+  // extract new claim ID from metadata
+  const nodes = (claimIdResult.result.meta as TransactionMetadata).AffectedNodes
+  const createdNodes = nodes.filter((node) =>
+    Object.keys(node).includes('CreatedNode'),
+  )
+  const claimIdsLedgerEntries = (createdNodes as CreatedNode[]).filter(
+    (node) => node.CreatedNode.LedgerEntryType === 'XChainOwnedClaimID',
+  )
+  if (claimIdsLedgerEntries.length !== 1) {
+    // This error should never hit and is just a sanity check
+    throw Error('Wallet has more than one claim ID')
+  }
+  const xchainClaimId = claimIdsLedgerEntries[0].CreatedNode.NewFields
+    .XChainClaimID as string
+
+  console.log(`Claim ID for the transfer: ${xchainClaimId}`)
+
+  // XChainCommit
+  console.log('Step 2: Locking the funds on the locking chain...')
+
+  const commitTx: XChainCommit = {
+    TransactionType: 'XChainCommit',
+    Account: wallet1.classicAddress,
+    Amount: xrpToDrops(1),
+    XChainBridge: bridge,
+    XChainClaimID: xchainClaimId,
+    OtherChainDestination: wallet2.classicAddress,
+  }
+  const commitResult = await lockingClient.submitAndWait(commitTx, {
+    wallet: wallet1,
+  })
+  console.log(commitResult)
+
+  // wait for the attestations to go through
+  ledgersWaited = 0
+  while (ledgersWaited < MAX_LEDGERS_WAITED) {
+    await sleep(LEDGER_CLOSE_TIME)
+    const currentBalance = await issuingClient.getXrpBalance(
```

Comment:
My guess is people are going to mess up which client to use to look up / submit transactions all the time. Is there a way to get nice errors if you try to submit a transaction from an account that doesn't exist on the network you're sending from? (Like "Account doesn't exist. If you're doing sidechain things, maybe you submitted to the wrong network?")

Reply:
Are you referring to specifically in this snippet or in the general error message? I can add it to the snippet, but I think it would be confusing to non-sidechain users to have it in the general error message.

Reply:
Maybe it'd be worth having a flag for "sidechain" clients which adds better error handlers for specific things? 

Ex. 
```
const client = new Client('url', sidechain=true)
```
Then behind the scenes:
```
client.on('error', () => {
 if(error is `ActNotFound`) {
   error.message += 'This may be caused by querying the wrong network. Searched <client.thisnetwork> for this account.
 }
}
```

(Not sure if there'd be that many of these changes, but if we get a couple that's one thought for how to make for a better debugging experience)

Reply:
Created an issue for it here: https://github.com/XRPLF/xrpl.js/issues/2484